projects/cpython3/gen_fuzz.py

```diff
@@ -0,0 +1,47 @@
+"""Generate a fuzz test.
+
+`gen_fuzz.py fuzz_test_file.inc fuzz_test_name` will output the fuzz test to stdout.
+"""
+import os
+import sys
+
+FUZZ_TEMPLATE = """
+#include <Python.h>
+#include <stdlib.h>
+#include <inttypes.h>
+
+%(fuzz_definition)s
+
+// CPython generates a lot of leak warnings for whatever reason.
+int __lsan_is_turned_off() { return 1; }
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  if (!Py_IsInitialized()) Py_InitializeEx(0);
+
+  int rv = %(fuzz_test_name)s((const char*)data, size);
+  if (PyErr_Occurred()) {
+    // Fuzz tests should handle expected errors for themselves.
+    PyErr_Print();
+    abort();
+  }
+  return rv;
```

Comment:
But I see that functions from https://github.com/ssbr/cpython/tree/fix-issue-29505/Modules/_fuzz only return 0, i.e. this is fine